### PR TITLE
[docs][orc8r] Document v1.3 upgrade procedure and touch up install docs.

### PR DESCRIPTION
**Pull request by xjtian**
Signed-off-by: Jacky Tian <xjtian@fb.com>

## Summary

- v1.2 -> v1.3 upgrade docs for Orchestrator

## Test Plan

- Deployed v1.2 cluster on AWS, upgraded it to v1.3 and sanity-checked functionality

## Additional Information

- [ ] This change is backwards-breaking

### CODE REVIEW

File: docs/readmes/orc8r/deploy_build.md [hunk 1]
```diff
@@ -77,14 +77,20 @@ To start, create a private GitHub repo to use as your Helm chart repo. We'll
 refer to this as `GITHUB_REPO`.
 
 Next, package the Magma Helm charts and publish them to the GitHub
-repo
+repo. You'll have to check out a temporary commit to build the `1.4.36` version
+of the orc8r chart for 1.3.x because `1.4.37` on the head of the release
+branch has some changes that you shouldn't try to deploy yet.
```
Comment (hcgatewood) [inline docs/readmes/orc8r/deploy_build.md:82]: What's up with the 1.4.36 vs. 1.4.37 messiness?

In any case, feels cleaner to put it as a `NOTE: if you're trying to package 1.4.37 ...`

File: docs/readmes/orc8r/deploy_install.md [hunk 2]
```diff
@@ -134,10 +135,14 @@ If you don't know what values to put for the `docker_*` and `helm_*` variables,
...
 
 Make sure that the `source` variables for the module definitions point to
-`github.com/magma/magma//orc8r/cloud/deploy/terraform/<module>`.
+`github.com/magma/magma//orc8r/cloud/deploy/terraform/<module>?ref=v1.3`.
 Adjust any other parameters as you see fit - check the READMEs for the
 relevant Terraform modules to see additional variables that can be set.
 
+The `?ref=v1.3` argument is *very* important to include in the Terraform
+`source` parameters for your modules - we will be making large sweeping changes
+to these modules on the master branch of magma in the near future.
```
Comment (hcgatewood) [inline docs/readmes/orc8r/deploy_install.md]: magma -> Magma

File: docs/readmes/orc8r/upgrade_1_3.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+hide_title: true
+---
+
+# Upgrade to v1.3
+
+This guide covers upgrading Orchestrator deployments from v1.2 to v1.3.
+Upgrades which skip versions (e.g. v1.1 to v1.3) are not explicitly documented
+or supported at this time.
+
+The v1.3 upgrade is pretty straightforward:
```
Comment (hcgatewood) [inline docs/readmes/orc8r/upgrade_1_3.md]: "The v1.3 upgrade follows a standard pattern" or something to that effect

File: docs/readmes/orc8r/upgrade_1_3.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
+---
+id: upgrade_1_3
+title: Upgrade to v1.3
+hide_title: true
+---
+
+# Upgrade to v1.3
+
+This guide covers upgrading Orchestrator deployments from v1.2 to v1.3.
+Upgrades which skip versions (e.g. v1.1 to v1.3) are not explicitly documented
```
Comment (hcgatewood) [inline docs/readmes/orc8r/upgrade_1_3.md:10]: Move this disclaimer to an intro page for the upgrades sub-section -- either that or add it as a disclaimer at the top of all the upgrade pages

**Review by hcgatewood — CHANGES_REQUESTED** [review]
Couple small things

File: docs/readmes/orc8r/deploy_build.md (hunk 1 above)
Comment (xjtian) [reply docs/readmes/orc8r/deploy_build.md:82]: Nobody should be trying to package 37 right now except us

**Comment by xjtian** [comment]
Merging after updates in the interest of time for 1.3.0 release

**Pull request merged**